Src/Plugins/Security/SEC1001.SecurePlaintextSecrets.json

```diff
@@ -222,6 +222,21 @@
           "SubId": "LinkedInSecretKey",
           "ContentsRegex": "(?i)linkedin(.{0,20})?[''\"](?<fingerprint>0-9a-z]{16})[''\"]",
           "MessageArguments": { "secretKind": "LinkedIn Secret Key" }
+        },
+        {
+          "SubId": "DynatraceToken",
+          "ContentsRegex": "\\b(?<fingerprint>dt0[a-zA-Z]{1}[0-9]{2}\\.[A-Z0-9]{24}\\.[A-Z0-9]{64})",
```

Comment:
>a-zA-Z [](start = 50, length = 6)

Invest a little normalizing this sort of this to our convention (numerics first, and use (?i) where possible